src/arch/mips/kernel/entry.S

```diff
@@ -21,13 +21,24 @@
  */
  LEAF(mips_second_exception_handler)
 	SAVE_ALL                             /* save all needed registers */
-
+	li $t1, 0x40000000                   /* 0x40000000 is a mask, which mean that Timer Interrupt bit of CAUSE register set in 1 */
+	and  $t1, $k1, $t1                   /* check if Timer Interrupt bit of CAUSE register is 1 */
+	bnez $t1, else                       /* if Timer Interrupt bit of CAUSE register is 1 then go else */
+	nop
+	andi  $k1, $k1, 0x7c                 /* read exception number */
```

Comment:
Can you please add spaces to align arguments like in other lines below?

Reply:
done